api/test/media_integration.py

```diff
@@ -227,6 +227,10 @@ def get_terms_set(res):
             or result["creator"] == item["creator"]
         ), f"{terms_set} {get_terms_set(result)}/{result['creator']}-{item['creator']}"
 
+        assert result["license_version"] is not None
+        assert result["attribution"] is not None
+        assert result["creator_url"] is not None
+
```

Comment:
This is a high priority PR that fixes an important issue, so I'm not proposing the change here, but I think we should use something like https://github.com/maticardenas/drf-openapi-tester/ to test that _all_ responses conform to the OpenAPI schema that we have. I'll open an issue for that.

Reply:
That is a great idea!